.github/workflows/tag_image_push.yml

```diff
@@ -14,111 +14,163 @@ on:
       RELEASE_VERSION:
         description: 'Version of the docker image e.g. 5.1.1, 5.1.1-1, defaults to HZ_VERSION'
         required: false
-      EDITIONS:
-        description: 'Editions to build'
+      RELEASE_TYPE:
```

Comment:
Do we not also need https://github.com/hazelcast/hazelcast-docker/pull/740?

Reply:
No, it was a failed idea and was replaced with `.github/release_type` file